entrypoint.sh

```diff
@@ -4,6 +4,11 @@
 # CLI arguments
 PY_VERSIONS=$1
 BUILD_REQUIREMENTS=$2
+SYSTEM_REQUIREMENTS=$3
+
+if [ ! -z "$SYSTEM_REQUIREMENTS" ]; then
+    yum install -y ${SYSTEM_REQUIREMENTS}
```

Comment:
Can you also add an error message and exit code if the yum install failed?
```suggestion
    yum install -y ${SYSTEM_REQUIREMENTS}  || { echo "Installing yum package(s) failed."; exit 1; }
```